preview/app/AppLoader.scala

```diff
@@ -80,8 +80,8 @@ trait AppComponents
   with OnwardServices
   with ApplicationsServices {
 
-  override lazy val capiHttpClient: HttpClient = wire[CapiHttpClient]
-  override lazy val contentApiClient = wire[ContentApiClient]
```

Comment:
the `preview` app was using its own `content.api.host` config item. It will now use the globally-defined preview host